Wordle.Api/Wordle.Api/Services/GameService.cs

```diff
@@ -68,11 +68,68 @@ public IQueryable<AllWordStats> StatsForAllWords()
 
         return result;
     }
-}
+
+    public async Task<GameStatsDto> WordOfDayStats(DateTime date, string? playerName)
+    {
+        DateOnly dateOnly = DateOnly.FromDateTime(date);
+
+        WordOfTheDay? word = await Db.WordsOfTheDays
+            .Include(wotd => wotd.Games)
+            .FirstOrDefaultAsync(wotd => wotd.Date == dateOnly);
+
+        IEnumerable<Game> wordOfTheDayGames;
+        GameStatsDto stats;
+
+        if (word is not null && word.Games.Count != 0)
```

Comment:
```suggestion
        if (word?.Games.Count != 0)
```
We can simplify this condition with the null conditional operator.